src/main.ts

```diff
@@ -59,11 +62,20 @@ async function run(): Promise<void> {
       pull_number: targetPullRequestNumber,
     });
 
+    const outputs = {
+      sha: pullRequest.data.head.sha,
+      branch: pullRequest.data.head.ref,
+      number: pullRequest.data.number,
+      pullRequest: JSON.stringify(pullRequest.data),
+    } as {[key: string]: string | number};
+
+    core.info('Set the following outputs:');
     // Can't use merge commit here because of `bin/bump-sentry`
-    core.setOutput('sha', pullRequest.data.head.sha);
-    core.setOutput('branch', pullRequest.data.head.ref);
-    core.setOutput('number', pullRequest.data.number);
-    core.setOutput('pullRequest', JSON.stringify(pullRequest.data));
+    for (const k of Object.keys(outputs)) {
+      const v = outputs[k];
```

Comment:
Can simplify this a bit:

```suggestion
    for (const [k, v] of Object.entries(outputs)) {
```